app/src/main/res/layout/fragment_shopinfo.xml

```diff
@@ -267,6 +267,31 @@
                     app:layoutManager="LinearLayoutManager"
                     app:recyclerAdapter="@{viewModel.adapter}"
                     />
+
+                <View
+                    android:layout_width="match_parent"
+                    android:layout_height="@dimen/dp_2"
+                    android:background="@color/color_gray_white"
+                    />
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:background="@color/color_gray_white"
+                    android:drawableStart="@drawable/ic_public"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:drawableLeft="@drawable/ic_public"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.